client/src/component/qa/Question.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from 'react';
+import Answer from './Answer.jsx';
+
+
+// handle single question
+const Question = ({productId, question}) => {
+  const [answers, showMoreAnswers] = useState(2);
+  const [expandAll, setExpanded] = useState(false);
+  const [helpful, markHelpful] = useState(false);
+
+  let allAnswers = Object.entries(question.answers).map((ans) => ans[1]);
+
+  let sortedAnswers = allAnswers.sort((a, b) => (b.helpfulness - a.helpfulness)).sort((a, b) => { if (a.answerer_name.toLowerCase() === 'seller') {
+    return -1;
+  } else if (b.answerer_name.toLowerCase() === 'seller') {
+    return 1;
+  }
+  });
+
+  const expandAnswers = () => {
+    if (expandAll) {
+      setExpanded(2);
+    } else {
+      setExpanded(sortedAnswers.length);
+    }
+    setExpanded(!expandAll);
+  };
+
```

Comment:
Hey ulan just making sure but is your eslint turned on and running with the airbnb style guide?